graf3d/eve7/inc/ROOT/REveDataCollection.hxx

```diff
@@ -101,6 +92,12 @@ public:
 
    using REveElement::GetHighlightTooltip;
    std::string GetHighlightTooltip(const std::set<int>& secondary_idcs) const override;
+
+   void SetItemsChangeDelegate (std::function<void (REveDataItemList*, const std::vector<int>&)> handler_func);
+   void SetFillImpliedSelectedDelegate (std::function<void (REveDataItemList*, Set_t& impSelSet)> handler_func);
```

Comment:
1. Please use typedefs for function types.
2. Use these typedefs also when declaring class members `_handler_items_change` and `_handler_fillimp`
3. Use default naming convention for these members like `fHandlerItemsChange`